infra/nixos/concrexit.nix

```diff
@@ -343,10 +343,26 @@ in
             "\"\"" = {
               default = !cfg.allowUnknownHost;
               locations."/".return = "444";
+              extraConfig = ''
+                access_log off;
+              '';
             };
           };
       };
 
+      logrotate = {
+        enable = true;
+        extraConfig = ''
+          /var/log/nginx/*.log {
+            daily
+            rotate 5
```

Comment:
Maybe 5 days is a bit too short? Technicie meets weekly, so having a 2 or at least 1 week log rotation doesn't sound weird to me (even though we hardly ever use them)

Reply:
I agree